terraform.tfstate

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
this file is not updated, either you have not used it after applying, this doesn't match backup file